sagemaker-python-sdk/keras_script_mode_pipe_mode_horovod/tensorflow_keras_CIFAR10.ipynb

```diff
@@ -0,0 +1,618 @@
+    "```python\n",
+    "distributions = {'mpi': {\n",
+    "                    'enabled': True,\n",
+    "                    'processes_per_host': # Number of Horovod processes per host\n",
+    "                        }\n",
+    "                }\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sagemaker.tensorflow import TensorFlow\n",
+    "\n",
+    "distributions = {'mpi': {\n",
+    "                    'enabled': True,\n",
+    "                    'processes_per_host': 4\n",
+    "                        }\n",
+    "                }\n",
+    "hyperparameters = {'epochs': 60, 'batch-size' : 256}\n",
+    "\n",
+    "source_dir = os.path.join(os.getcwd(), 'source_dir')\n",
+    "estimator_dist = TensorFlow(base_job_name='dist-cifar10-tf',\n",
+    "                       entry_point='cifar10_keras_main.py',\n",
+    "                       source_dir=source_dir,\n",
+    "                       role=role,\n",
+    "                       framework_version='1.12.0',\n",
+    "                       py_version='py3',\n",
+    "                       hyperparameters=hyperparameters,\n",
+    "                       train_instance_count=2, train_instance_type='ml.p3.8xlarge',\n",
+    "                       tags = [{'Key' : 'Project', 'Value' : 'cifar10'},{'Key' : 'TensorBoard', 'Value' : 'dist'}],\n",
+    "                       metric_definitions=keras_metric_definition,\n",
+    "                       distributions=distributions)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this example, we'll configure ```wait=False``` if you want to see the output logs change to ```wait=True```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "remote_inputs = {'train' : inputs+'/train', 'validation' : inputs+'/validation', 'eval' : inputs+'/eval'}\n",
+    "estimator_dist.fit(remote_inputs, wait=False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Local TensorBoard command\n",
+    "Using TenosrBoard we can compare between the different jobs we ran, the following command will print the tensorboard command.\n",
+    "Run it in any environment where you have TensorBoard installed.  "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!python generate_tensorboard_command.py"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can install [TensorBoard](https://github.com/tensorflow/tensorboard) locally using `pip install tensorboard`\n",
+    "To access an S3 log dir, you need to configure TensorBoard default region, you can do that by configuaring an environment variable AWS_REGION=region_name, e.g. `export AWS_REGION = 'us-east-1'`\n",
```

Comment:
"To access an S3 log directory, configure the TensorBoard default region. You can do this by configuring an environment variable named AWS_REGION, and setting the value of the environment variable to the AWS region your training jobs run in. For example, `export AWS_REGION = 'us-east-1'`\n",